config/vendor-api-v1.yml

```
@@ -7,7 +7,7 @@ info:
     name: DfE
     email: becomingateacher@digital.education.gov.uk
   description: |
-    API for DfE's Apply for postgraduate teacher training service.
+    API for DfE’s Apply for postgraduate teacher training service.
```

Might as well fix this while we’re here

```suggestion
    API for DfE’s Apply for teacher training service.
```